### ISSUE TITLE
No option to disable LFS, causing giant sdist tarball

### ISSUE THREAD

**Issue by pllim** [OpenAstronomy/github-actions-workflows#168]
https://github.com/OpenAstronomy/github-actions-workflows/blob/59a929bdf64d072b19c8fc758ae44dd768f6f3fe/.github/workflows/publish_pure_python.yml#L111

I have a package with some test data stored under Git LFS.  But I never want to package them for distribution. They are just for dev testing. There does not seem to be an option to not grab LFS in this workflow, causing my sdist tarball to blow up from 100 kB or so to 1.1 GB.

**Comment by astrofrog** [comment]
What data is included in the source distribution is something that should be configurable via setuptools or the MANIFEST?

**Comment by pllim** [comment]
Sure. In the end, it is easier for me to write my own workflow to not call the LFS checkout and be done with it. Though I guess if I ever have time to revisit, I will have to hunt down all the files and update the manifest properly. If you think `lfs` should always be `true`, then you can close this as won't fix. Thanks for the quick reply!

**Comment by Cadair** [comment]
Yeah I would say this is primarily a problem with your packaging, either in manifest or in pyproject.toml you should be able to exclude these files. Now there is an argument for an option to turn off lfs to save checkout time, but there are already too many options 😉
